### DISCUSSION TITLE
Custom remote control UI for client

### DISCUSSION THREAD

**Discussion by Anutrix** [rustdesk/rustdesk#2739] [General]
For example, I have a comic reader app that changes pages using arrow keys.
An option for a custom remote design generation with common controls like Arrow Keys on the Android client would be nice.

We can further improve it by allowing users can design custom remotes with shortcuts, common actions like minimize, alt+tab, etc. too.

**Comment by rustdesk** [comment]
https://github.com/users/rustdesk/projects/12/views/1
